ui/src/language/json/de.json

```diff
@@ -9,21 +9,21 @@
   "download-computer": "Auf den Computer herunterladen",
   "download-to-server": "Auf den Server herunterladen",
   "episode-downloaded-to-server": "Episode auf den Server heruntergeladen",
-  "cpu-info": "CPU Info",
+  "hardware": "Hardware",
   "cpu-brand": "CPU Marke",
   "cpu-cores": "CPU Kerne",
   "podcast-size": "Podcast Verzeichnis Größe",
-  "cpu-usage": "CPU-Auslastung",
+  "cpu-usage": "CPU",
   "used-cpu": "Verwendete CPU",
   "free-cpu": "Freie CPU",
-  "memory-usage": "Speicherauslastung",
+  "memory-usage": "Speicher",
   "used-memory": "Verwendeter Speicher",
   "loading": "Laden",
   "free-memory": "Freier Speicher",
-  "disk-usage": "Festplattenauslastung",
+  "disk-usage": "Scheibe",
```

Comment:
Should be Festplattenauslastung. Scheibe means Slice in English.

Reply:
Oh, haha, thanks. Google Translate 🤣